### DISCUSSION TITLE
How to pass context to Schema.from_orm()?

### DISCUSSION THREAD

**Discussion by yshalsager** [vitalik/django-ninja#929] [Q&A] [answered]
In a Schema model that have resolver method, context can be accessed if the queryset is returned from a view function directly, but not if I serialize manually using `.from_orm()`.

Is there's a way to pass context while using `from_orm()`?

**Comment by vitalik** [comment, marked as answer]
Hi @yshalsager 

Please upgrade to latest (1.0.1) - to pass context:

```Python
SomeSchema.from_orm(obj, context={'foo': 'bar'}
```

**Reply by yshalsager** [reply]
Thanks for the amazing speed and great help @vitalik !
